<?xml version="1.0" encoding="UTF-8"?>

<package name="javax.management" release="1">
    <class name="RuntimeErrorException">
        <constructor params="java.lang.Error">
            <RegionEffects>none</RegionEffects>
            <Starts>nothing</Starts>
            <Unique>return</Unique>
        </constructor>

        <constructor params="java.lang.Error, java.lang.String">
            <RegionEffects>none</RegionEffects>
            <Starts>nothing</Starts>
            <Unique>return</Unique>
        </constructor>

        <method name="getCause">
            <Borrowed>this</Borrowed>
            <RegionEffects>reads Instance</RegionEffects>
            <Starts>nothing</Starts>
        </method>

        <method name="getTargetError">
            <Borrowed>this</Borrowed>
            <RegionEffects>reads Instance</RegionEffects>
            <Starts>nothing</Starts>
        </method>
    </class>
</package>
